### ISSUE TITLE
Question about Validation

### ISSUE THREAD

**Issue by xizhu1022** [huangtinglin/NGCF-PyTorch#15]
According to the original paper, the authors randomly selected 10% of interactions as validation set to tune hyper-parameters. However, it seems no validation set is constructed to select models and the best performance on the test set is finally reported in your implementation. 
Looking forward to your reply.

**Comment by huangtinglin** [comment]
Sorry for the late reply. I have not implemented the evaluation of the model on the validation set, which is inconsistent with the setting described in the paper. But the official implementation applies the same strategy...I will take time to reproduce the evaluation on the validation set. Thanks for your suggestion.
